target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -77,6 +77,10 @@ tplink,re650-v1)
 xzwifi,creativebox-v1)
 	ucidef_set_led_netdev "internet" "internet" "$boardname:blue:internet" "wan"
 	;;
+xiaomi,mi-router-ac2100)
+	ucidef_set_led_switch "wan-yellow" "WAN (yellow)" "$boardname:yellow:wan" "switch0" "0x01" "0x01"
```

Comment:
Could you explain why speed_mask is set to 0x01 here? What is the expected behavior?

Reply:
Inteded usage of the speed masks at the WAN LED's definition:
If a cable is connected but the speed can't be recognized the LED will be yellow. If a cable is connected and the speed can be recognized (10mbps, 100mbps, 1000mbps) the light will be blue. By this you can easily get a hint on cabling problems.

Reply:
`ucidef_set_led_switch` is obsolete in DSA driver. It won't work at all.

Please take a look at [MI R3G's dts](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_xiaomi_mir3g.dts) and use speed/netdev LED trigger instead.